.travis.yml

```diff
@@ -30,35 +30,34 @@ env:
         - CONDA_CHANNELS='astropy-ci-extras astropy'
 
     matrix:
-        # Make sure that egg_info works without dependencies
+
         - SETUP_CMD='egg_info'
-        # Try all python versions with the latest numpy
-        - SETUP_CMD='test'
 
 matrix:
     include:
 
-        # Do a coverage test in Python 2.
-        - python: 2.7
+        # Try Astropy/NumPy development versions. This requires them to be
+        # compiled during setup which takes some time.
+        - python: 3.5
+          env: ASTROPY_VERSION=development NUMPY_VERSION=dev
```

Comment:
But: fun hackday project is to write a bot that opens temporary PRs to given packages when the RC comes out to test for any braking codes. It's a bit messy as the PRs shouldn't be merged but closed, but doing it once in every half year may be worth it. 
I wonder what @astrofrog and @eteq think about it?